presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableWriterNode.java

```diff
@@ -100,7 +97,7 @@ public PlanNode getSource()
         return source;
     }
 
-    @JsonProperty
+    @JsonIgnore
```

Comment:
prevents the writer target from being serialized.  We don't want to send it to workers, since we're not rewriting e.g. InsertReference to InsertHandle.  It's not needed for execution, and can be expensive to serialize (e.g. `CreateName` has the column comments in the ConnectorTableMetadata).

Reply:
@rschlussel : So technically other places that we use optional trick to avoid json serialization can also use this annotation? ;)

Reply:
Yes!